src/drawDirective.coffee

```diff
@@ -25,12 +25,13 @@ angular.module('ui-leaflet')
     scope: false
     replace: false
     require: ['leaflet']
-    controller: ($scope) ->
+    controller: ['$scope', ($scope) ->
```

Comment:
It would help if you could try bumping that and seeing if the issue resolves itself. If not I will merge this NP.